app/models/patient.rb

```diff
@@ -10,6 +10,28 @@ class Patient < ApplicationRecord
   include ActiveModel::Validations
   include FhirHelper
 
+  MONITORING_REASONS = [
+    'Not Monitoring',
```

Comment:
Where did 'Not Monitoring' come from?

Reply:
The Validation Helper. I will remove it from the list.

Reply:
Where?
https://github.com/SaraAlert/SaraAlert/blob/master/app/helpers/validation_helper.rb#L87

Reply:
good point. I must've added it in as a test-field when wiring everything up, and forgotten to remove it. It's been removed from this PR.